pscheduler-core/pscheduler-core/bash_completion

```diff
@@ -0,0 +1,158 @@
+# pscheduler(1) completion                                 -*- shell-script -*-
+
+_pscheduler_base()
+{
+    local base
+
+    if [ -d /usr/libexec/pscheduler ]; then
+        base=/usr/libexec/pscheduler
+    else
+        base=/usr/lib/pscheduler
+    fi
+
+    echo $base
+}
+
+_pscheduler_classes()
+{
+    ls "$(_pscheduler_base)/classes" | sed 's/$/s/'
+}
+
+_pscheduler_tools()
+{
+    ls "$(_pscheduler_base)/classes/tool"
+}
+
+_pscheduler_tests()
+{
+    ls "$(_pscheduler_base)/classes/test"
+}
+
+_pscheduler_tasks()
+{
+    if type curl &>/dev/null; then
+        curl -s -k https://localhost/pscheduler/tasks | tr -d ',"[]'
+    fi
+}
+
+_pscheduler()
+{
+    local cur prev words cword
+    _init_completion -s || return
+
+    # handle specific options
```

Comment:
Many of these go with the `task` command; some are for specific tests.

Reply:
It does not matter really, if you enter `--ip-version` anywhere this will offer you `4` and `6` as the next word.